chart/compass/values.yaml

```diff
@@ -1,6 +1,7 @@
 global:
   disableLegacyConnectivity: true
   defaultTenant: 3e64ebae-38b5-46a0-b1ed-9ccee153a0ae
+  fooTenant: 1eba80dd-8ff6-54ee-be4d-77944d17b10b
```

Comment:
What's that tenant for? Just curious

Reply:
This tenant is used to run the System Broker tests. We considered running them in a separate tenant to ensure that we System Broker tests won't mess up Director or Connector e2e tests when they're all run concurrently.

Reply:
We've also undone this change in favor of using the default tenant for the e2e tests. We've decided to be consistent with the other component. Also, there's no chance of a race condition between all e2e tests because they're not executed concurrently at the moment. 😆